operator/pkg/reconciliation/reconcile_racks.go

```diff
@@ -2181,15 +2366,6 @@ func (rc *ReconciliationContext) ReconcileAllRacks() (reconcile.Result, error) {
 		return result.Error(err).Output()
 	}
 
-	// We do the node and pvc taint checks here
-	// with the assumption that the cluster is healthy
-
-	if utils.IsPSPEnabled() {
-		if err := rc.checkNodeAndPvcTaints(); err != nil {
-			return result.Error(err).Output()
-		}
-	}
```

Comment:
Since we are now permitting not ready pods on nodes undergoing EMM, this check has been moved.